lib/plist-transform.js

```diff
@@ -2,9 +2,8 @@
 const path = require('path');
 const fs = require('fs');
 const plist = require('plist');
-const pify = require('pify');
-
-const fsP = pify(fs);
+const read = require('read-glob-promise');
```

Comment:
It's battle tested in production and very fast. It being mine is not really the point here. I would always prefer a module I've used before and trust. No, it doesn't read the file contents, but that requires just a few characters extra characters, and it's kinda weird to mix those concerns.